dkg/bcast/server.go

```diff
@@ -92,18 +102,32 @@ func (s *server) dedupHash(pID peer.ID, msgID string, hash []byte) error {
 	return nil
 }
 
-func (s *server) handleSigRequest(_ context.Context, pID peer.ID, m proto.Message) (proto.Message, bool, error) {
+func (s *server) handleSigRequest(ctx context.Context, pID peer.ID, m proto.Message) (proto.Message, bool, error) {
 	req, ok := m.(*pb.BCastSigRequest)
 	if !ok {
 		return nil, false, errors.New("invalid message type")
 	}
 
+	fn, found := s.getMessageIDFunc(req.Id)
+	if !found {
+		return nil, false, errors.New("unknown message id", z.Str("message_id", req.Id))
+	}
+
+	reqMessageHash, err := s.hashFunc(req.Message)
+	if err != nil {
+		return nil, false, errors.Wrap(err, "hash any")
```

Comment:
Weird unclear error message.

Reply:
That's the standard way we do errors: https://github.com/ObolNetwork/charon/blob/main/docs/goguidelines.md#error-handling

The wrapped error will eventually have enough context, the user-facing error will be much clearer and contain more context.

Reply:
My concern was about the wording "hash any" that does not make a big sense to me..